srcpkgs/go-ipfs/files/ipfs/log/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec vlogger -t ipfs
```

Comment:
the run file can be replaced with a link to `/usr/bin/vlogger`, vlogger will get the tag from the service name.

Reply:
Done